solutions/arrays/rotate-array-clockwise.js

```diff
@@ -0,0 +1,24 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let arr=readLine().split(" ")
+let rotate=parseInt(readLine())
+while(rotate>0)
+{
+	let shifted=arr.shift()

```

Comment:
@LVSsandeepVarma , is the concept of arr.shift() will be completed before giving this problem? if not we need to change the logic of the problem

Reply:
@rpapavan97  yes we have covered it